src/strings/normalize.c

```diff
@@ -507,6 +521,8 @@ static MVMint32 is_spacing_mark(MVMThreadContext *tc, MVMCodepoint cp) {
     }
 }
 static MVMint32 should_break(MVMThreadContext *tc, MVMCodepoint a, MVMCodepoint b) {
+    int GCB_a = MVM_unicode_codepoint_get_property_int(tc, a, MVM_UNICODE_PROPERTY_GRAPHEME_CLUSTER_BREAK);
+    int GCB_b = MVM_unicode_codepoint_get_property_int(tc, b, MVM_UNICODE_PROPERTY_GRAPHEME_CLUSTER_BREAK);
```

Comment:
I would like to call `MVM_unicode_codepoint_get_property_int` before `switch (GCB_a) ` and `switch (GCB_b)` until it is used, which may avoids two calls for fast-path check.